### PR TITLE
FCCEUD-14: Describe modules

### PR DESCRIPTION
**Pull request by theashiot**
JIRA ID: FCCEUD-14

JIRA Link: https://projects.engineering.redhat.com/browse/FCCEUD-14

* Create concept module "Modules"
* Create concept module "Comparison between modules and topics in traditional
  topic-based authoring"

Link to analysis doc: https://docs.google.com/document/d/1mKO-81RpTHdSoy1SpehWsSjners6GZnf16zzfXBIGwc/edit?usp=sharing

### CODE REVIEW

**Comment by benradey** [comment]
Oops, didn't notice this was a draft until after I approved it. Feel free to continue making commits, we'll re-review when this is ready.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_comparison-between-modules-and-topics.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+include::../attributes.adoc[]
+[id='comparison-between-modules-and-topics_{context}']
+= Comparison between modules and topics in traditional topic-based authoring
+
+A module, in {RH}'s modular documentation approach, is similar to a topic in traditional topic-based authoring. The differences between the two are in scope, and application - whereas a topic is a general and abstract concept, a module is a concrete implementation of the abstract concept specific to modular documentation, similar to how a DITA topic defined by the OASIS group is a specific implementation of a topic.
+
+For a topic-based content to be a module, the content must conform to the definition a module as described in Modules.
```
Comment (robin-owen) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_comparison-between-modules-and-topics.adoc]: Rewriting to remove some of the repetition and the (in my opinion, unnecessary) comma. Is this still accurate?

"To be considered a module, topic-based content must conform to the definition of a module."

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+include::../attributes.adoc[]
+[id='module_{context}']
+= Module
```
Comment (robin-owen) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module.adoc]: In my opinion, this title doesn't really tell us what the concept being explained is.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+The following are the properties of a module:
+
+* Conforms to the module syntax.
+* Self-contained: A module contains all the necessary information about a topic required by a user to achieve a goal. The goal of a user can be to understand a concept, to perform a task, or to find reference information.
+* Standalone: A module does not require any surrounding text to be useful to the reader; a module is publishable on its own.
+* Pluggable: Amodule can be used unmodified as a part of a larger document.
+
+Types of modules defined in modular documentation:
+
+* Concept module
```
Comment (robin-owen) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module.adoc]: We already stated these are types of modules, so we should be fine to just list the type (concept, procedure, reference).

**Comment by theashiot** [comment]
Thanks @robin-owen for the feedback! 

Working on incorporating the suggestions.

best,
ashwin

**Comment by theashiot** [comment]
Hi @robin-owen, I've made the suggested changes to "module" module. Also, I've re-written the other module. Do you think there are more differences between modules and topics? And also, should we elaborate on how the structure differs (once we have frozen a module structure)?

best,
ashwin

**Pull request merged**